lib/sitemap.ts

```diff
@@ -1,9 +1,9 @@
-import { promises } from 'node:fs';
+import { writeFile } from 'node:fs/promises';
```

Comment:
@B2o5T I use the following next library on my personal website to generate an accurate sitemap: https://github.com/iamvishnusankar/next-sitemap

The sitemap below does not include the blog posts links